Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/LargeDayRangeDemoViewController.swift

```diff
@@ -24,12 +24,26 @@ final class LargeDayRangeDemoViewController: DemoViewController {
     super.viewDidLoad()
 
     title = "Large Day Range"
+  }
+
+  override func viewWillLayoutSubviews() {
+    super.viewWillLayoutSubviews()
+
+    guard !didScrollToInitialMonth else { return }
+
+    let padding: CGFloat
+    switch monthsLayout {
+    case .vertical: padding = calendarView.layoutMargins.top
+    case .horizontal: padding = calendarView.layoutMargins.left
+    }
 
     let january1500CE = calendar.date(from: DateComponents(era: 1, year: 1500, month: 01, day: 01))!
     calendarView.scroll(
       toMonthContaining: january1500CE,
-      scrollPosition: .firstFullyVisiblePosition,
+      scrollPosition: .firstFullyVisiblePosition(padding: padding),
```

Comment:
this is just improving the example project behavior a bit. This ensures that the month we scroll to won't be obscured by any bars.